src/loadgenerator/requirements.txt

```diff
@@ -63,7 +63,7 @@ six==1.16.0
     #   geventhttpclient
 typing-extensions==4.5.0
     # via locust
-urllib3==1.26.15
+urllib3==2.0.2
```

Comment:
We already got this update through https://github.com/GoogleCloudPlatform/microservices-demo/pull/1797.
Closing this pull-request.